linux_os/guide/system/software/updating/clean_components_post_updating/ansible/shared.yml

```diff
@@ -11,6 +11,16 @@
     option: solver.upgradeRemoveDroppedPackages
     value: true
     create: False
+{{% elif 'ubuntu' in product %}}
+- name: Ensure Apt Removes Previous Package Versions
```

Comment:
Please, ensure the [Project Style Guides](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible)